frontend/src/components/cytology/CytologyDashBoard.js

```diff
@@ -330,7 +338,18 @@ function CytologyDashboard() {
                 name="statusFilter"
                 labelText={intl.formatMessage({ id: "label.filters.status" })}
                 value={
-                  filters.statuses.length > 1 ? "All" : filters.statuses[0].id
+                  filters.statuses.length === 3 &&
```

Comment:
we can also avoid hardcoding a specific length here ie` 3` , by simply checking  if length > 1 and the list  includes "COMPLETED" status